components/postgresql/actions/execute-custom-query/execute-custom-query.mjs

```diff
@@ -0,0 +1,23 @@
+import postgresql from "../../postgresql.app.mjs";
+
+export default {
+  name: "Execute Custom Query",
+  key: "postgresql-execute-custom-query",
+  description: "Executes a custom query you provide. [See Docs](https://node-postgres.com/features/queries)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    postgresql,
+    query: {
+      propDefinition: [
+        postgresql,
+        "query",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const res = await this.postgresql.executeQuery(this.query);
```

Comment:
Not sure if this will be secured against sql injection, since is not using any kind of [prepared statements](https://node-postgres.com/features/queries#prepared-statements) here. I would suggest to add support for prepared statemens so if you want you can take a look at this example with [MySql execute-query.mjs](https://github.com/PipedreamHQ/pipedream/blob/ac90bf536ce0e911ac0bc7b07d6ba2e23b09e18e/components/mysql/actions/execute-query/execute-query.mjs#L52)